packages/grafana-ui/src/components/uPlot/config/UPlotAxisBuilder.ts

```diff
@@ -43,6 +43,15 @@ export interface AxisProps {
 }
 
 export const UPLOT_AXIS_FONT_SIZE = 12;
+
+// for panels with small heights, we increase tick density by reducing the minumum tick spacing
+// all values in CSS pixels
+const Y_TICK_SPACING_PANEL_HEIGHT = 150;
+const Y_TICK_SPACING_NORMAL = 30;
+const Y_TICK_SPACING_SMALL = 15;
```

Comment:
Hey, works perfectly!
Really minor question (you can ignore it 😄 ), maybe we can stick to some standard space system (scale system) you know, like following a ratio like 8px (8, 16, 24, 32, 48, etc.)
Or is it some meaning behind having 15, 30, 150 values?

Reply:
it was 30 originally, so i just divided by two. as long as it's bigger than the font height plus some extra spacing it should be okay. the font size is 12.

anyways these are just _minimum_ spacings. the actual spacing adapts to the vertical panel height, so there's no benefit in using "standard" sizes, imo.